src/battle-tooltips.ts

```diff
@@ -1912,6 +1909,10 @@ class BattleTooltips {
 					if (ally !== pokemon) {
 						value.modify(1.3, 'Power Spot');
 					}
+				} else if (allyAbility === 'Steely Spirit') {
+					if (move.type === 'Steel') {
+						value.modify(1.5, 'Steely Spirit');
+					}
```

Comment:
```suggestion
				} else if (moveType === 'Steel' && allyAbility === 'Steely Spirit') {
					value.modify(1.5, 'Steely Spirit');
```

Reply:
several of the surrounding ifs are nested , i will just go ahead and fix those as well (in the morning im so tired)